Tiltfile

```diff
@@ -359,7 +359,9 @@ def deploy_worker_templates(template, substitutions):
         calico_values = "./templates/addons/calico/values.yaml"
     flavor_cmd += "; " + helm_cmd + " repo add projectcalico https://projectcalico.docs.tigera.io/charts; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install calico projectcalico/tigera-operator -f " + calico_values + " --namespace tigera-operator --create-namespace; kubectl --kubeconfig ./${CLUSTER_NAME}.kubeconfig apply -f ./templates/addons/calico/felix-override.yaml"
     if "external-cloud-provider" in flavor_name:
-        flavor_cmd += "; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install --repo https://raw.githubusercontent.com/kubernetes-sigs/cloud-provider-azure/master/helm/repo cloud-provider-azure --generate-name --set infra.clusterName=${CLUSTER_NAME}"
+        # TODO: remove extra config once the azure-cloud-provider Helm chart release supports VMSS Flex.
```

Comment:
Cloud-provider-azure released v1.26 last week so this should only be a matter of updating the Helm chart https://github.com/kubernetes-sigs/cloud-provider-azure/releases/tag/v1.26.0

cc @lzhecheng

Reply:
That release did not update the Helm chart, so provisioning with Kubernetes v1.26.0 gives an error about `imageTag required` without this workaround.

Reply:
Right, I'm saying we should update the Helm chart (@lzhecheng @jackfrancis are usually the ones who maintain it but anyone can make a PR).

Reply:
See kubernetes-sigs/cloud-provider-azure#2994